app/workers/crawler/geocoding.rb

```diff
@@ -0,0 +1,33 @@
+require 'rexml/document'
+class Crawler::Geocoding < Crawler::Base
+  include Crawler::Searchable
+  
+  def initialize
+    super
+    @file_path = "#{Rails.root}/app/workers/crawler/"
+    @places = JSON.parse(File.read(@file_path + 'my_basket.json'))
+  end
+
+  def run
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for run is too high. [21.61/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Method has too many lines. [20/10] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)